.github/CODEOWNERS

```diff
@@ -1,11 +1,29 @@
 # See https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners
 
-*                                         @DataDog/apm-dotnet
-.github/workflows/profiler-pipeline.yml   @DataDog/profiling-dotnet
-/shared/                                  @DataDog/apm-dotnet @DataDog/profiling-dotnet
+# By default every team is owner
+*                                         @DataDog/apm-dotnet @DataDog/debugger-dotnet @DataDog/profiling-dotnet
+
+# Tracer
 /tracer/                                  @DataDog/apm-dotnet
+
+# CI
 /tracer/src/Datadog.Trace/Ci/             @DataDog/apm-dotnet @DataDog/ci-app-tracers
-# /tracer/src/Datadog.Trace/AppSec/       @DataDog/apm-dotnet @DataDog/appsec-dotnet
+/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/   @DataDog/ci-app-tracers
+
+# ASM
+/tracer/src/Datadog.Trace/AppSec/         @DataDog/apm-dotnet @DataDog/asm-dotnet
+/tracer/test/test-applications/security   @DataDog/apm-dotnet @DataDog/asm-dotnet
+/tracer/test/Datadog.Trace.Security.IntegrationTests @DataDog/apm-dotnet @DataDog/asm-dotnet
+/tracer/test/Datadog.Trace.Security.Unit.Tests       @DataDog/apm-dotnet @DataDog/asm-dotnet
+
+# Profiler
 /profiler/                                @DataDog/profiling-dotnet
+.github/workflows/profiler-pipeline.yml   @DataDog/profiling-dotnet
```

Comment:
is it possible to add  `/tracer/src/Datadog.Trace/ContinuousProfiler` please ?

Reply:
✅